package.json

```diff
@@ -14,11 +14,11 @@
   "proxy": "http://localhost:4142/",
   "scripts": {
     "build": "npm run build:css && react-scripts build",
-    "build:css": "node-sass src/ -o src/",
+    "build:css": "sass --no-source-map src/",
     "postbuild": "rm -rf build/api",
     "start": "REACT_APP_DATA_SOURCE=$DATA npm-run-all -p start:css start:app start:lib",
     "start:app": "PORT=4141 react-scripts start",
-    "start:css": "npm run build:css && node-sass src/ -o src/ --watch --recursive",
+    "start:css": "npm run build:css && sass --no-source-map --watch style.scss:style.css",
```

Comment:
It seems something has changed here that doesn't compile the styles on the fly when after running `npm start`. I checked out your branch locally, ran it, made a style change, but didn't see anything reflected in the browser. I had to kill the process and then run `npm start` again to see my change. 

Are you seeing the same thing.

Reply:
let me check, when you said you had to kill the process, do you mean you stopped and restarted the BE?

Reply:
No, just the frontend. I killed the `npm start` process.

Reply:
hey @tynandebold great spot thank you. I updated the script to watch all the changes inside src/, it works fine from my side locally. How about yours now?

Reply:
i dont know much about the shell script but i removed `--recursive` from the original script as it wasn't compatible with dart-scss, Should this be an issue in any case? @rashidakanchwala @tynandebold @cvetankanechevska ?

Reply:
Very nice, it works well for me now.

For `--recursive` it looks like this may just work out of the box now. And since we're watching the whole `src/.` directory it seems to be working as it should.